deploy.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+
+# TeslaMate MCP Deployment Script
+# This script helps manage the Docker deployment of TeslaMate MCP
+
+set -e
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+# Function to print colored output
+print_status() {
+    echo -e "${GREEN}[INFO]${NC} $1"
+}
+
+print_error() {
+    echo -e "${RED}[ERROR]${NC} $1"
+}
+
+print_warning() {
+    echo -e "${YELLOW}[WARNING]${NC} $1"
+}
+
+# Check if Docker and Docker Compose are installed
+check_requirements() {
+    print_status "Checking requirements..."
+    
+    if ! command -v docker &> /dev/null; then
+        print_error "Docker is not installed. Please install Docker first."
+        exit 1
+    fi
```

Comment:
```suggestion
    if ! which docker >& /dev/null; then
        print_error "Docker is not installed. Please install Docker first."
        exit 1
    fi
```

could it be that the operator is incorrectly defined in this control mechanism? when I tested it, it gave me an error even though I have a docker cli tool. we can proceed with `which` for control.